lte/gateway/c/core/oai/tasks/mme_app/experimental/mme_app_serialization.cpp

```diff
@@ -0,0 +1,297 @@
+/**
+ * Copyright 2021 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//--C includes -----------------------------------------------------------------
```

Comment:
For this and other files on the PR please run clang-format to fix most of the linting issues